README.md

```diff
@@ -1,41 +1,13 @@
 # FortiGate Autoscale with AWS Transit Gateway Integration
 
-## Project archive notice
-**This project has been archived.** it is recommended to use the most current version of the **Amazon AWS** auto scaling deployment, located in the project [FortiGate Autoscale for AWS](https://github.com/fortinet/fortigate-autoscale-aws). This project supports any combination of Bring Your Own License (BYOL) and On-Demand instances as well as the option for Transit Gateway integration.
+This repository has been archived.
 
-## Introduction
-FortiGate Autoscale is collection of **Node.js** modules and cloud-specific templates which support auto scaling functionality for groups of FortiGate-VM instances on various cloud platforms.
+The ***FortiGate Autoscale with AWS Transit Gateway Integration*** project has been replaced by [https://github.com/fortinet/fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws). The Transit Gateway integration feature provided here in this project is also available in the new [fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws) project.
```

Comment:
```suggestion
The ***FortiGate Autoscale with AWS Transit Gateway Integration*** project has been moved to a new location: [https://github.com/fortinet/fortigate-autoscale-aws](https://github.com/fortinet/fortigate-autoscale-aws).
```

Reply:
When users click the URL, they will see Transit Gateway mentioned in the About:
AWS Transit Gateway can be used to connect Amazon Virtual Private Clouds (VPCs) and their on-premises networks to a single gateway. FortiGate Autoscale with Transit Gateway integration extends the protection to all networks connected to the Transit Gateway.